test-specifications/token-service/tokenCreateTransaction.md

```diff
@@ -0,0 +1,1154 @@
+| 2       | Creates a token with an expiration time of one less than the current time       | name="testname", symbol="testsymbol", treasuryAccountId=<OPERATOR_ACCOUNT_ID>, expirationTime=<CURRENT_TIME> - 1       | The token creation fails with an INVALID_EXPIRATION_TIME response code from the network.                                  | N                 |
+| 3       | Creates a token with an expiration time 8,000,002 seconds from the current time | name="testname", symbol="testsymbol", treasuryAccountId=<OPERATOR_ACCOUNT_ID>, expirationTime=-8000002                 | The token creation fails with an INVALID_EXPIRATION_TIME response code from the network.                                  | N                 |
+
+#### JSON Request Example
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 12,
+  "method": "createToken",
+  "params": {
+    "name": "testname",
+    "symbol": "testsymbol",
+    "treasuryAccountId": "0.0.2",
+    "expirationTime": 5184000
+  }
+}
+```
+
+#### JSON Response Example
+
+```json
+{
+  "jsonrpc": "2.0",
+  "id": 12,
+  "result": {
+    "tokenId": "0.0.541",
+    "status": "SUCCESS"
+  }
+}
+```
+
+### **Auto Renew Account:**
+
+- The ID of the account to pay for the auto-renewal of the token.
+
+| Test no | Name                                                          | Input                                                                                                                  | Expected response                                                                          | Implemented (Y/N) |
+|---------|---------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------|-------------------|
+| 1       | Creates a token with an auto renew account                    | name="testname", symbol="testsymbol", treasuryAccountId=<OPERATOR_ACCOUNT_ID>, autoRenewAccountId=<VALID_ACCOUNT_ID>   | The token creation succeeds and the token has a valid auto-renew account.                  | N                 |
+| 2       | Creates a token with an auto renew account that doesn't exist | name="testname", symbol="testsymbol", treasuryAccountId=<OPERATOR_ACCOUNT_ID>, autoRenewAccountId="123.456.789"        | The token creation fails with an INVALID_AUTORENEW_ACCOUNT response code from the network. | N                 |
+| 3       | Creates a token with an empty auto renew account              | name="testname", symbol="testsymbol", treasuryAccountId=<OPERATOR_ACCOUNT_ID>, autoRenewAccountId=""                   | The token creation fails with an SDK internal error.                                       | N                 |
```

Comment:
Yes, it would throw an error in this case. But written as "empty auto renew account" can be interpreted as just missing auto renew account, and not "invalid auto renew account"

Reply:
I can update the language here to something like "blank auto renew account", or maybe "empty auto renew account string"?

Reply:
Perhaps "Creates a token with the auto renew account not set"?

Reply:
^ I'm fine with this is @0xivanov is good with it

Reply:
I'm Ok

Reply:
Updated